manual_test/coverage_local_jacocorunner/test.sh

```diff
@@ -24,7 +24,8 @@ test_coverage_with_local_jacocorunner() {
 }
 
 build_local_jacocorunner() {
-    $root_dir/scripts/build_jacocorunner/build_jacocorunner_bazel_5.0+.sh
+    # Build for Bazel 6 (as this is the default Bazel version in the repo).
+    $root_dir/scripts/build_jacocorunner/build_jacocorunner_bazel_5.0+.sh 6
```

Comment:
Sine this script now takes a bazel version as a parameter, maybe you could remove version from file name? I think it would be a bit cleaner.

Reply:
I mean, there are two scripts. One for Bazel 4.* and another for Bazel 5+ (5 and 6).
Should I remove the Bazel 4.* script maybe? (or rename it to the 4.0 suffix)

Reply:
And then this new script can have a name without the version suffix.

Reply:
The issue was that the script for Bazel 4.* is significantly different (including comments), so I was hesitant to unite the script for the three Bazel versions supported.

Reply:
I'll rename the script for Bazel 4.* to have a version suffix in the filename, and rename the Bazel 5.0+6.0 script to be the "default" (without version suffix). It should be more obvious this way (especially as Bazel 6.0 is the minimum supported version in rules_scala currently).

Reply:
Done.